### DISCUSSION TITLE
사이트가 동작하지 않습니다.

### DISCUSSION THREAD

**Discussion by 99999kcal** [List-KR/community#49] [Q&A]
https://filters.adtidy.org/extension/chromium/filters/227.txt 이걸 adguard ios의 DNS 필터에 추가해서 사용하는데, 이걸 활성화 하면 zdnet 사이트가 동작하지 않습니다.

**Comment by piquark6046** [comment]
그걸 DNS 블록리스트로 사용하시면 안됩니다. 혹시 나무위키를 보시다가 추가하셨다면 나무위키를 끄시고 먼저 README.md를 읽어보시길 바랍니다.
https://github.com/List-KR/List-KR#%EC%82%AC%EC%9A%A9%EB%B2%95

**Reply by piquark6046** [reply]
![image](https://github.com/List-KR/community/assets/98787049/5db6a018-9623-401f-ab7f-7d5da6574cca)

**Reply by piquark6046** [reply]
@99999kcal

**Reply by 99999kcal** [reply]
친절한 답 정말 감사합니다..!
